### ISSUE TITLE
Cannot configure multiple servers in OPENVPN_CONFIG

### ISSUE THREAD

**Issue by g000444555** [haugene/docker-transmission-openvpn#2266]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

version: '3.8'

services:
  transmission-openvpn:
    cap_add:
      - NET_ADMIN
    volumes:
      - "${HOME}/yr2/appdata/samba/downloads/:/data"
    environment:
      - PUID=${UID}
      - PGID=${GID}
      - OPENVPN_PROVIDER=NORDVPN
      # For OPENVPN_CONFIG values refer to https://nordvpn.com/ovpn/
      - OPENVPN_CONFIG=uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com
      - OPENVPN_USERNAME=secret
      - OPENVPN_PASSWORD=secret
      - NORDVPN_COUNTRY=GB
      - LOCAL_NETWORK=10.0.44.0/24
      - TRANSMISSION_SPEED_LIMIT_UP=5
      - TRANSMISSION_SPEED_LIMIT_UP_ENABLED=true
      - TRANSMISSION_SPEED_LIMIT_DOWN=20000
      - TRANSMISSION_SPEED_LIMIT_DOWN_ENABLED=false
      - TRANSMISSION_RATIO_LIMIT=0.2000
      - TRANSMISSION_RATIO_LIMIT_ENABLED=true
    logging:
      driver: json-file
      options:
        max-size: 10m
    ports:
      - '1238:9091'
    image: haugene/transmission-openvpn
    deploy:
      restart_policy:
        condition: on-failure
        delay: 5s
        max_attempts: 3
        window: 120s

### Current Behavior

When I configure multiple servers via the env variable OPENVPN_CONFIG, the container startup fails with the following error:

```
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Best server : uk2290.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading config: default.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk2290.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Using OpenVPN CONFIG :: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading config: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 3 servers found in OPENVPN_CONFIG, fr-uk16.nordvpn.com chosen randomly
appconf-transmission-openvpn-1  | Supplied config fr-uk16.nordvpn.com.ovpn could not be found.
appconf-transmission-openvpn-1  | Your options for this provider are:
appconf-transmission-openvpn-1  | default.ovpn
appconf-transmission-openvpn-1  | uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | NB: Remember to not specify .ovpn as part of the config name.
appconf-transmission-openvpn-1 exited with code 1
```

### Expected Behavior

From: https://haugene.github.io/docker-transmission-openvpn/run-container/ Pro tip: choose multiple servers. For example: OPENVPN_CONFIG=france,sweden,austria,italy,belgium This will ensure a location near you, but at the same time it will allow some redundancy. Set Docker to restart the container automatically and you have a failover mechanism. The container chooses one of the configs at random when it starts and it will bounce from server to server until it finds one that works.

### How have you tried to solve the problem?

When I use a single configuration like OPENVPN_CONFIG=uk1814.nordvpn.com the container starts up

### Log output

```
appconf-transmission-openvpn-1  | Starting container with revision: 94754fe91d3a430bafa6b0dec75f56ded43ef1df
appconf-transmission-openvpn-1  | Creating TUN device /dev/net/tun
appconf-transmission-openvpn-1  | Using OpenVPN provider: NORDVPN
appconf-transmission-openvpn-1  | Running with VPN_CONFIG_SOURCE auto
appconf-transmission-openvpn-1  | Provider NORDVPN has a bundled setup script. Defaulting to internal config
appconf-transmission-openvpn-1  | Executing setup script for NORDVPN
appconf-transmission-openvpn-1  | Downloading user specified config. NORDVPN_PROTOCOL is set to: UDP
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Checking curl installation
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Removing existing configs
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Selecting the best server...
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Searching for country : GB (227)
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Searching for group: legacy_p2p
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Searching for technology: openvpn_udp
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Best server : uk2290.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Downloading config: default.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk2290.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Using OpenVPN CONFIG :: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Downloading config: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:31 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 3 servers found in OPENVPN_CONFIG, uk1814.nordvpn.com chosen randomly
appconf-transmission-openvpn-1  | Supplied config uk1814.nordvpn.com.ovpn could not be found.
appconf-transmission-openvpn-1  | Your options for this provider are:
appconf-transmission-openvpn-1  | default.ovpn
appconf-transmission-openvpn-1  | uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | NB: Remember to not specify .ovpn as part of the config name.
appconf-transmission-openvpn-1 exited with code 1
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Selecting the best server...
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Searching for country : GB (227)
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Searching for group: legacy_p2p
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Searching for technology: openvpn_udp
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Best server : uk2290.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Downloading config: default.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk2290.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Using OpenVPN CONFIG :: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Downloading config: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:33 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 3 servers found in OPENVPN_CONFIG, uk1814.nordvpn.com chosen randomly
appconf-transmission-openvpn-1  | Supplied config uk1814.nordvpn.com.ovpn could not be found.
appconf-transmission-openvpn-1  | Your options for this provider are:
appconf-transmission-openvpn-1  | default.ovpn
appconf-transmission-openvpn-1  | uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | NB: Remember to not specify .ovpn as part of the config name.
appconf-transmission-openvpn-1 exited with code 1
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Checking curl installation
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Removing existing configs
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Selecting the best server...
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Searching for country : GB (227)
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Searching for group: legacy_p2p
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Searching for technology: openvpn_udp
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Best server : uk2290.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading config: default.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk2290.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Using OpenVPN CONFIG :: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading config: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:35 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 3 servers found in OPENVPN_CONFIG, fr-uk16.nordvpn.com chosen randomly
appconf-transmission-openvpn-1  | Supplied config fr-uk16.nordvpn.com.ovpn could not be found.
appconf-transmission-openvpn-1  | Your options for this provider are:
appconf-transmission-openvpn-1  | default.ovpn
appconf-transmission-openvpn-1  | uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | NB: Remember to not specify .ovpn as part of the config name.
appconf-transmission-openvpn-1 exited with code 1
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Selecting the best server...
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Searching for country : GB (227)
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Searching for group: legacy_p2p
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Searching for technology: openvpn_udp
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Best server : uk2290.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Downloading config: default.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk2290.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Using OpenVPN CONFIG :: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Downloading config: uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | 2022-05-21 11:17:37 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.udp.ovpn
appconf-transmission-openvpn-1  | 3 servers found in OPENVPN_CONFIG, fr-uk21.nordvpn.com chosen randomly
appconf-transmission-openvpn-1  | Supplied config fr-uk21.nordvpn.com.ovpn could not be found.
appconf-transmission-openvpn-1  | Your options for this provider are:
appconf-transmission-openvpn-1  | default.ovpn
appconf-transmission-openvpn-1  | uk1814.nordvpn.com,fr-uk16.nordvpn.com,fr-uk21.nordvpn.com.ovpn
appconf-transmission-openvpn-1  | NB: Remember to not specify .ovpn as part of the config name.
appconf-transmission-openvpn-1 exited with code 1

```

### HW/SW Environment

```markdown
- OS: Arch Linux with kernel 5.17.3-arch1-1
- Docker: Docker version 20.10.14


docker info
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc., v0.8.1-docker)
  compose: Docker Compose (Docker Inc., 2.4.1)

Server:
 Containers: 4
  Running: 3
  Paused: 0
  Stopped: 1
 Images: 18
 Server Version: 20.10.14
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: false
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc io.containerd.runc.v2 io.containerd.runtime.v1.linux
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: de8046a5501db9e0e478e1c10cbcfb21af4c6b2d.m
 runc version: 
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
  cgroupns
 Kernel Version: 5.17.3-arch1-1
 Operating System: EndeavourOS
 OSType: linux
 Architecture: x86_64
 CPUs: 16
 Total Memory: 62.18GiB
 Name: papaya
 ID: CPEU:LKNF:Q3WM:P2GX:H7RN:F6PO:6XDU:JXMP:LJKN:VOHE:NXS4:QD3T
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

```
```


### Anything else?

_No response_

**Comment by edgd1er** [comment]
Nordvpn has a specific config script, this type of options is not supported at the moment.

https://haugene.github.io/docker-transmission-openvpn/provider-specific/

the dev tag image has a new feature for nordvpn provider, you can set the server. #2142 
2022/01/04: add NORDVPN_SERVER to download the config file based on server's fqdn

**Comment by g000444555** [comment]
Thank you for the very quick response and for the NORDVPN_SERVER configuration hint. 

I ended up commenting out the configuration `OPENVPN_CONFIG`, because the script seems to query and find the best server. For example, refer to the log output `Best server : uk1891.nordvpn.com`.

On the other hand, when I tried to use `OPENVPN_CONFIG` with a single server's fqdn, the vpn connection quickly failed and did not fall back to anything. I think the `Best server` query along with auto-restart configuration in docker-compose offer a good  failover mechanism almost as the one described in the documentation snippet above.

I will try the NORDVPN_SERVER config option if I encounter further issues, but for now I will leave my config as it is, as long as it's working. Thanks again. Cheers.
